nss-day-cohort-30/bangazon-site-cunning-carps
# Cancel Order

#Description
Previously, a user was not able to cancel or "delete" an order. This PR adds a cancel order button that removes the order and the order products.

#Type of change
New feature (non-breaking change which adds functionality)
 
#Testing Instructions for Change Made
git fetch --all
git checkout OrderCancelButton
dotnet run
verify...

#Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings
 I have added test instructions that prove my fix is effective or that my feature works
